checkers/ruleguard_checker.go

```diff
@@ -34,62 +35,89 @@ func init() {
 
 func newRuleguardChecker(info *linter.CheckerInfo, ctx *linter.CheckerContext) *ruleguardChecker {
 	c := &ruleguardChecker{ctx: ctx}
-	rulesFilename := info.Params.String("rules")
-	if rulesFilename == "" {
+
+	var rules []string
+	if info.Params.IsStringSlice("rules") {
+		rules = info.Params.StringSlice("rules")
+	} else if info.Params.IsString("rules") {
+		rules = []string{info.Params.String("rules")}
+	} else {
 		return c
 	}
 
+	for _, rulesFileName := range rules {
+		if rulesFileName == "" {
+			continue
+		}
+		rset, err := parseRuleGuardFile(rulesFileName)
+		if err != nil {
+			log.Printf("unable to parse file %s: %+v", rulesFileName, err)
+			return c
+		}
+		c.rset = append(c.rset, goRuleSetWithFileName{
+			RuleFileName: rulesFileName,
+			GoRuleSet:    rset,
+		})
+	}
+	return c
+}
+
+func parseRuleGuardFile(rulesFilename string) (*ruleguard.GoRuleSet, error) {
 	// TODO(quasilyte): handle initialization errors better when we make
 	// a transition to the go/analysis framework.
 	//
 	// For now, we log error messages and return a ruleguard checker
 	// with an empty rules set.
-
 	data, err := ioutil.ReadFile(rulesFilename)
 	if err != nil {
-		log.Printf("ruleguard init error: %+v", err)
-		return c
+		return nil, fmt.Errorf("ruleguard init error: %+v", err)
```

Comment:
looks like a good place for %w :)

Reply:
You are right, I wasn't sure if go-critic uses go 1.13